users/factories.py

```diff
@@ -33,6 +33,12 @@ class Meta:
         model = User
 
 
+class StaffUserFactory(UserFactory):
```

Comment:
We don't need this factory since we will use the fixture as mentioned in above comment.